src/Aspire.Hosting.PostgreSQL/PostgresBuilderExtensions.cs

```diff
@@ -400,7 +400,13 @@ public static IResourceBuilder<PostgresServerResource> WithInitBindMount(this IR
         ArgumentNullException.ThrowIfNull(builder);
         ArgumentException.ThrowIfNullOrEmpty(source);
 
-        return builder.WithBindMount(source, "/docker-entrypoint-initdb.d", isReadOnly);
+        const string initPath = "/docker-entrypoint-initdb.d";
+
+        var importFullPath = Path.GetFullPath(source, builder.ApplicationBuilder.AppHostDirectory);
+
+        return builder.WithContainerFiles(
```

Comment:
This is one change I'm on the fence about; I'd argue using the `cp` functionality is better for all these init scripts, but a handful of resources named the init methods `WithInitBindMount`. Do we want to make the method inaccurately named and update it to use `WithContainerFiles` or introduce a new `WithInitFiles` method and mark the `WithInitBindMount` method as deprecated?